src/openrct2-ui/windows/TrackDesignManage.cpp

```diff
@@ -139,7 +139,7 @@ static void WindowTrackManageMouseup(rct_window* w, rct_widgetindex widgetIndex)
         case WIDX_RENAME:
             WindowTextInputRawOpen(
                 w, widgetIndex, STR_TRACK_DESIGN_RENAME_TITLE, STR_TRACK_DESIGN_RENAME_DESC, {},
-                _trackDesignFileReference->name, 127);
+                _trackDesignFileReference->name.c_str(), 127);
```

Comment:
Seems 127 is the maximum amount of filename characters? Could you make that into a `constexpr` while we’re modifying this line?